chipy_org/apps/meetings/templates/meetings/emails/rsvp_email.html

```diff
@@ -1 +1,24 @@
-To change your rsvp, click <a href="http://{{ site.domain }}{% url 'update_rsvp_with_key' key %}">here</a>
+Your {{ rsvp.get_venue_display }} RSVP for {{ rsvp.meeting }} has been {{ rsvp.get_status_display }}.
+
+{% if rsvp.status == "A" %}
```

Comment:
Seems like this should have more conditions based on attendance. If they are attending in-person this makes sense, but there's a condition missing here for the others.

This is all about de-incentivizing folks from taking slots, so the check is rather complicated. Are they going in-person? If not, does it have a virtual event limit? If either of the previous questions then display. And waitlists fill up too, so waitlists for either might be included.

Reply:
I read this a couple time.  I think I get it.  

Basically you're saying we should **not** de-incentivized people if the meeting has no capacity limit.  For example, we would probably not set a limit for virtual meeting so, telling them to update their RSVP really isn't necessary.  

The wait lists won't fill up.  Anyone not accepted will be wait listed.  Though they could be so far down the list they practically would never get a spot.

Reply:
But if someone decides to never attend and they are on the waitlist shouldn't they update their status when it's convenient rather than risking them not doing it once they get a slot?

The idea is that you want folks to take action if their action will improve the experience for themselves or others. The simple answer here is that action isn't based on whether they are `confirmed` (A) or `waitlisted` but rather whether  their reservation would block another's involvement on the night.

Reply:
Well that is basically everyone who has not declined their RSVP.  So, I'll just remind folks to please update their status if they change their mind regardless.  

The thing to do here is to estimate how many no shows we typically have and then overbook accordingly.   I don't have a clue what that percentage would be or how to gauge the enthusiasm for a first in-person chipy meeting in two years.